docs/common/gradle-common-groovy-properties.md

```diff
@@ -16,10 +16,11 @@
     // Optionally specify schema dependencies on other gradle projects
     dependency project(':OtherProject')
 
-    // The dialect version you would like to target
-    // Defaults to "sqlite:3.18"
+    // The dialect version you would explicitly like to target. This can be skipped
+    // on Android where a SQLite version is automatically picked based on your minSdk.
+    // Defaults to "sqlite:3.18".
```

Comment:
Maybe "Otherwise defaults to..." since this sentence and the previous one are somewhat in disagreement.

Reply:
done!